shared/nm-utils/tests/test-shared-general.c

```diff
@@ -0,0 +1,77 @@
+/* -*- Mode: C; tab-width: 4; indent-tabs-mode: t; c-basic-offset: 4 -*- */
```

Comment:
A nit: could we perhaps stop adding these?
A source file is *not* the right place to configure anyone's editor.

Reply:
Done. Plus added check to `checkpatch.pl`. ACK that too?